### PR TITLE
Add support for the --enable-llvm-debug  flag in kompile

### PR DESCRIPTION
**Pull request by anvacaru**
part of: https://github.com/runtimeverification/evm-semantics/issues/1670

### CODE REVIEW

File: src/pyk/ktool/kompile.py [hunk 1]
```diff
@@ -109,6 +111,7 @@ def kompile(
         ccopts=ccopts,
         no_llvm_kompile=no_llvm_kompile,
         concrete_rules=concrete_rules,
+        enable_llvm_debug=enable_llvm_debug,
```
Comment (tothtamas28) [inline src/pyk/ktool/kompile.py:114]: Consider ordering arguments in the order of parameters in the function signature.

File: src/pyk/ktool/kompile.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def kompile(
     ccopts: Iterable[str] = (),
     no_llvm_kompile: bool = False,
     enable_search: bool = False,
+    enable_llvm_debug: bool = False,
```
Comment (tothtamas28) [inline src/pyk/ktool/kompile.py:59]: In all related function definitions and function calls, consider ordering arguments in the order of parameters established here.

**Review by tothtamas28 — APPROVED** [review]

File: src/pyk/ktool/kompile.py (hunk 2 above)
Comment (ehildenb) [reply src/pyk/ktool/kompile.py:59]: @anvacaru this was not addressed. Either respond to the comment, or address the issue.

Also, I believe that you missed some spots where you need to thread the argument through to. In this PR, for example (https://github.com/runtimeverification/pyk/pull/375/files), I had to thread it through to 8 places. In your PR, I only see it theraded through to 6 places.

Comment (anvacaru) [reply src/pyk/ktool/kompile.py:59]: My bad, I'll open a new PR to address this.

**Pull request merged**